Unable to compile from source with Java 18

# Problem
When attempting to compile from source using the commands listed in the README, I get the following error: `Unrecognized VM option 'MaxPermSize=512M'` (full error message below).

Java 18 is the default in the latest Ubuntu LTS release.

## Error message
```sh
$ ./gradlew build
Downloading https://services.gradle.org/distributions/gradle-7.4-bin.zip
...........10%...........20%...........30%...........40%...........50%...........60%...........70%...........80%...........90%...........100%
Starting a Gradle Daemon (subsequent builds will be faster)

FAILURE: Build failed with an exception.

* What went wrong:
Unable to start the daemon process.
This problem might be caused by incorrect configuration of the daemon.
For example, an unrecognized jvm option is used.
Please refer to the User Manual chapter on the daemon at https://docs.gradle.org/7.4/userguide/gradle_daemon.html
Process command line: /usr/lib/jvm/java-18-openjdk-amd64/bin/java -XX:MaxPermSize=512M -XX:+HeapDumpOnOutOfMemoryError --add-opens java.base/java.util=ALL-UNNAMED --add-opens java.base/java.lang=ALL-UNNAMED --add-opens java.base/java.lang.invoke=ALL-UNNAMED --add-opens java.base/java.util=ALL-UNNAMED --add-opens java.prefs/java.util.prefs=ALL-UNNAMED --add-opens java.prefs/java.util.prefs=ALL-UNNAMED --add-opens java.base/java.nio.charset=ALL-UNNAMED --add-opens java.base/java.net=ALL-UNNAMED --add-opens java.base/java.util.concurrent.atomic=ALL-UNNAMED -Xmx4g -Dfile.encoding=UTF-8 -Duser.country=US -Duser.language=en -Duser.variant -cp /home/vagrant/.gradle/wrapper/dists/gradle-7.4-bin/c0gwcg53nkjbqw7r0h0umtfvt/gradle-7.4/lib/gradle-launcher-7.4.jar org.gradle.launcher.daemon.bootstrap.GradleDaemon 7.4
Please read the following process output to find out more:
-----------------------
Unrecognized VM option 'MaxPermSize=512M'
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
```

# Proposed solution
I found this option in gradle.properties and simply removed it to get past this error message.

# Proposed enhancement
If certain modules are missing, the error message (from gradle?) it great about telling the user what the issue is, but is not as helpful in telling the user what commands to run to resolve it. I propose adding a note that explains this.

Error message:
```
FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':setupwizard:compileReleaseJavaWithJavac'.
> Failed to install the following Android SDK packages as some licences have not been accepted.
     platforms;android-32 Android SDK Platform 32
     build-tools;30.0.3 Android SDK Build-Tools 30.0.3
```

Solution:  
```sh
$ANDROID_HOME/cmdline-tools/bin/sdkmanager --install --sdk_root=$ANDROID_HOME "platforms;android-32"
$ANDROID_HOME/cmdline-tools/bin/sdkmanager --install --sdk_root=$ANDROID_HOME "build-tools;30.0.3"
```

--- a/README.md
+++ b/README.md
@@ -57,11 +57,19 @@ Code has also been contributed by:
 Build
 =====
 
-```
+```sh
 git clone --depth 1 https://github.com/mtotschnig/MyExpenses.git
 cd MyExpenses
 export ANDROID_HOME={sdk-dir}
 ./gradlew build
+```
+
+If gradlew gives you a "Failed to install the following Android SDK packages"
+error message, the packages can be installed manually with commands such as:
+
+```sh
+$ANDROID_HOME/cmdline-tools/bin/sdkmanager --install --sdk_root=$ANDROID_HOME "platforms;android-32"
+$ANDROID_HOME/cmdline-tools/bin/sdkmanager --install --sdk_root=$ANDROID_HOME "build-tools;30.0.3"
 ```
 
 Integrate

--- a/README.md
+++ b/README.md
@@ -72,6 +72,10 @@ $ANDROID_HOME/cmdline-tools/bin/sdkmanager --install --sdk_root=$ANDROID_HOME "p
 $ANDROID_HOME/cmdline-tools/bin/sdkmanager --install --sdk_root=$ANDROID_HOME "build-tools;30.0.3"
 ```
 
+If gradlew errors out with "Could not dispatch a message to the daemon", just
+re-run the command. This can happen when the system is low on memory. The same
+is true for the "Gradle build daemon disappeared unexpectedly" error.
+
 Integrate
 =========
 My Expenses now has experimental support for inserting data from third party apps. See [TransactionsContract.java](https://github.com/mtotschnig/MyExpenses/blob/master/transactionscontract/src/main/java/org/totschnig/myexpenses/contract/TransactionsContract.java).

--- a/gradle.properties
+++ b/gradle.properties
@@ -1,3 +1,3 @@
 android.useAndroidX=true
 android.enableJetifier=true
-org.gradle.jvmargs=-Xmx4g -XX:MaxPermSize=512M -XX:+HeapDumpOnOutOfMemoryError
+org.gradle.jvmargs=-Xmx4g -XX:+HeapDumpOnOutOfMemoryError

